spec/spec_helper_acceptance.rb

```diff
@@ -7,6 +7,7 @@
   install_package(host, 'redis-tools')
 end
 
-require 'beaker/module_install_helper'
-install_module_on(hosts)
-install_module_dependencies_on(hosts)
+# require 'beaker_puppet_helpers'
+install_puppet_module_via_pmt
```

Comment:
@ekohl Could you give me a clue as to what the modern way of installing modules and their dependencies on all hosts is?

Reply:
```
# frozen_string_literal: true

require 'voxpupuli/acceptance/spec_helper_acceptance'

configure_beaker do |host|
  install_puppet_module_via_pmt_on(host, 'puppetlabs-apache')
end
```

this will install the module itself, all deps from metadata.json + puppetlabs-apache.